### DISCUSSION TITLE
Acceleration Issue

### DISCUSSION THREAD

**Discussion by bemidji3** [prl-mushr/mushr#17] [Q&A]
My research partner and I have been building a mushr.io robot as a part of a class that we are taking this semester, and we have recently completed the assembly of the robot itself.  We have observed some very wonky behavior with the acceleration of the robot when we are using the Playstation Dualshock wireless controller to drive it.  We took a video of the behavior and have included a link to it in this post. 

https://user-images.githubusercontent.com/50583904/117185366-c15a5b00-ada7-11eb-823c-c2a8298feb41.mov

Please note that we have the latest version of the vesc code pulled and have run catkin_make with the latest code.  We also do not observe this behavior when we use the BLDC tool to operate the robot, this only occurs when we operate it with the playstation dualshock controller.  Any advice/help/ideas about this issue would be greatly appreciated.

**Comment by bemidji3** [comment]
> My research partner and I have been building a mushr.io robot as a part of a class that we are taking this semester, and we have recently completed the assembly of the robot itself. We have observed some very wonky behavior with the acceleration of the robot when we are using the Playstation Dualshock wireless controller to drive it. We took a video of the behavior and have included a link to it in this post.
> 
>  img-0827_su1ZIi2W.mov 
> Please note that we have the latest version of the vesc code pulled and have run catkin_make with the latest code. We also do not observe this behavior when we use the BLDC tool to operate the robot, this only occurs when we operate it with the playstation dualshock controller. Any advice/help/ideas about this issue would be greatly appreciated.

Looks to me like the video is not appearing correctly.  I have attached a zip of the original to this post for better access.

[IMG_0827.MOV.zip](https://github.com/prl-mushr/mushr/files/6429157/IMG_0827.MOV.zip)

**Reply by schmittlema** [reply]
Hi Bemidji3,
When was the lastime you pulled [vesc](https://github.com/prl-mushr/vesc)? I found a bug this morning that may relate to your issue, could you repull vesc, and rerun `catkin_make`?
